spec/unit/ssl/certificate_request_spec.rb

```diff
@@ -92,14 +92,13 @@
 
     it "should set the subject to [CN, name]" do
       request.generate(key)
-      # OpenSSL::X509::Name only implements equality as `eql?`
-      expect(request.content.subject).to eql OpenSSL::X509::Name.new([['CN', key.name]])
+      expect(request.content.subject).to eq OpenSSL::X509::Name.new([['CN', key.name]])
```

Comment:
So that remark was left in https://github.com/puppetlabs/puppet/commit/97c8b0c5eb95cee1bb08571512b258179d61da9f#diff-dc34e6762244dae4d2f0c66adfbe4cd8R93

stdlib docs mention this about `eql` - https://ruby-doc.org/stdlib-2.0.0/libdoc/openssl/rdoc/OpenSSL/X509/Name.html#method-i-eql-3F

> Returns true if name and other refer to the same hash key.

Clearly the tests pass and you mention the reason in the commit msg - just making sure you had reviewed the previous remarks.